static/settings.ts

```diff
@@ -229,6 +235,8 @@ export class Settings {
         this.fillColourSchemeSelector(this.root.find('.colourScheme'), this.settings.theme);
         this.setSettings(this.settings);
         this.handleThemes();
+
+        this.eventHub.on('settingsChange', this.onSettingsChange.bind(this));
```

Comment:
This is now needed to be able to disable the slider based on the checkbox right? What was the behaviour without this line?

Reply:
Exactly - without this onSettingsChange was called only during `Settings` construction. I didn't look too closely, just matched the usage for other `onSettingsChange`s.

Reply:
This is certainly doable without subscribing to the event, just wanted to use this chance to make `onSettingsChange` used in the uniform way.